spec/models/manageiq/providers/ibm_cloud/vpc/cloud_tools_spec.rb

```diff
@@ -146,15 +163,44 @@ def create_bearer_info(valid_expire_time: true, set_api_key: true)
     end
   end
 
+  # Test that the VPC client can be instantiated.
   it 'can get a VPC client' do |example|
     VCR.use_cassette("#{described_class.name.underscore}/#{example.description}") do
       expect(described_class.new(:api_key => api_key).vpc(:region => 'us-east').client).to be_a(IbmVpc::VpcV1)
     end
   end
 
+  # Test that a initialized VPC class can refresh its token if expired.
+  it 'can get a VPC client with expired auth' do |example|
+    VCR.use_cassette("#{described_class.name.underscore}/#{example.description}") do
```

Comment:
Not for this PR, but there is a way to do this part automatically, I believe.  If I recall, you can do the following:

```suggestion
  it 'can get a VPC client with expired auth', :with_vcr => true do
```

This wraps the entire test in a well-defined name based on the example description

Reply:
I was playing with that when I created this the first time. The directory structure wasn't being created correctly. I can't remember what was actually generated. I can play with it later.

Reply:
> The directory structure wasn't being created correctly.

I wonder if the issue is this spec is in `spec/models` but the code lives in `lib/` not `app/models`

Reply:
If I remember correctly is was creating new directories for each namespace in the class hierarchy.

Reply:
Yeah, it's possible it won't line up exactly with what you have now, but it might be cleaner to just follow the convention.

Reply:
Hah. I'm all for using the `vcr` metadata. That's what I used to configure my tests in the SDK.  I was just following the convention used by the other tests because I thought that was the standard for ManageIQ. If we want to convert to the metadata format we should update all the tests in this repo to use the same method.

Reply:
I don't disagree :)